### ISSUE TITLE
Forum (zelf gepost)

### ISSUE THREAD

**Issue by brussee** [csrdelft/csrdelft.nl#117]
Ik zie maar 1 post per forum-deel.

**Comment by brussee** [comment]
Tevens zie ik geen eigen posts in besloten forumdelen.

**Comment by thervh70** [comment]
@brussee is dit nog steeds van toepassing? Zo ja, kun je dan iets meer details geven over dit probleem? Ik snap het namelijk niet.

**Comment by brussee** [comment]
Ja, zie het verschil tussen forum recent op profiel vs zijbalk kopje.

**Comment by thervh70** [comment]
Hmm interessant.

**Comment by brussee** [comment]
Forum profiel toont meerdere berichten per draad en forum zelf zijbalk zou 1 per draad moeten tonen, maar doet nu 1 per forum deel zo lijkt het en prive forumdelen ontbreken in het geheel.

**Comment by M2WZ** [comment]
De zijbalk is niet meer, en op iemands profiel wordt correct gefilterd op draadjes waar de ingelogde wel/niet bij kan. Deze issue is dus niet meer relevant.
